### DISCUSSION TITLE
普通版本如何进行缓冲时间控制

### DISCUSSION THREAD

**Discussion by ronyongxian** [bytedance/xgplayer#863] [Q&A]
**您使用的西瓜播放器版本是多少？ What version of xgplayer are you using?**
unpkg.byted-static.com/xgplayer/2.31.2/browser/index.js

**您使用的操作系统和浏览器分别是？ What OS and browser are you using?**

**如何复现问题？ What did you do?**

**您期望的播放器正常行为是？ What did you expect to happen?**
Player可以进行缓冲时间控制，目前貌似默认是1分钟？

**实际播放器的表现是？ What actually happened?**

**可填写您所在的公司和相关产品业务，方便我们提供更好的技术支持 You can write your company and product which uses xgplayer,  for helping us provide better technical support.**

**Comment by gemxx** [comment]
使用MediaElement source播放无法控制buffer缓冲长度，这些都是浏览器行为。仅能通过 [preload](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-preload) 属性控制是否加载

**Comment by hughfenghen** [comment]
不确定想要控制buffer 长度的具体场景是啥。
像直播场景实时拉流为了尽量降低延迟， 可能会动态调整video播放速度倍率来控制buffer 长度。
